packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterPanel.tsx

```diff
@@ -74,14 +98,24 @@ export function GridFilterPanel() {
     [apiRef],
   );
 
+  React.useEffect(() => {
+    if (
+      linkOperators.length > 0 &&
+      filterModel.linkOperator &&
+      !linkOperators.includes(filterModel.linkOperator)
+    ) {
+      applyFilterLinkOperator(linkOperators[0]);
+    }
+  }, [linkOperators, applyFilterLinkOperator, filterModel.linkOperator]);
+
   React.useEffect(() => {
     if (items.length > 0) {
       lastFilterRef.current!.focus();
     }
   }, [items.length]);
 
   return (
-    <GridPanelWrapper>
+    <GridPanelWrapper sx={sx}>
```

Comment:
We usually pass to the root all the remaining props.

https://github.com/mui/mui-x/blob/2bc8e32a4d52cdd89fb2903bce0cf8ee535267fd/packages/grid/_modules_/grid/components/menu/GridMenu.tsx#L101